### PR TITLE
Adding Fabric Website

### PR DESCRIPTION
**Pull request by battletoilet**
#### Description of changes

Adding fabric website to repo and added to rush.json.

#### Focus areas to test

The fabric website.

### CODE REVIEW

**Comment by micahgodbolt** [comment]
Are there easy steps for us to test all of this?

File: packages/fabric-website/.gitignore [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Windows image file caches
```
Comment (cliffkoh) [inline packages/fabric-website/.gitignore]: remove this entire file

File: common/temp_modules/rush-fabric-website/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "gutil": "1.6.4",
+    "highlight.js": "9.6.0",
+    "mocha": "~2.5.3",
+    "orchestrator": "^0.3.7",
+    "react-addons-test-utils": "^15.1.0",
+    "react-highlight": "0.8.0",
+    "source-map-loader": "0.1.5",
+    "tslint": "^3.15.1",
+    "tslint-microsoft-contrib": "^2.0.9",
+    "typescript": "^2.0.6",
```
Comment (cliffkoh) [inline common/temp_modules/rush-fabric-website/package.json]: updating to appropriate versions of web-library-build inherently has typescript 2.1 (which the main office ui fabric react package is using as well). 

recommend having all packages be on same version of build tools and thus on same version of typescript for compilation.

File: packages/fabric-website/gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+/** @todo: disable lint config. */
+build.tslint.setConfig({
+  lintConfig: require('./tslint.json'),
+  sourceMatch: ['src/**/*.ts', 'src/**/*.tsx', '!src/**/*.scss.tsx']
+});
+
+/* Configure TypeScript 2.0. */
+build.typescript.setConfig({ typescript: require('typescript') });
```
Comment (cliffkoh) [inline packages/fabric-website/gulpfile.js]: Get rid of this if you're just using the version of web-library-build that comes with Typescript 2.1

Comment (dzearing) [inline packages/fabric-website/gulpfile.js]: I think you should keep it there; however in your package.json make sure it is using the same typescript version as every project.

**Pull request merged**